src/screens/Login/index.tsx

```diff
@@ -1,15 +1,123 @@
-import { Box, Column, Heading, Center } from 'native-base';
+import { Box, Heading, Center, FormControl, Input, Button, VStack, HStack, Link, Text } from 'native-base';
+import { useForm, Controller } from 'react-hook-form';
+import { useState } from 'react';
+import { useNavigation } from '@react-navigation/native';
+import { useLoginMutation, AuthUserInput } from '../../generated/graphql';
+import { TouchableOpacity } from 'react-native';
```

Comment:
Lets use the Button component from native-base instead of TouchableOpacity from react-native